src/App.scss

```
@@ -1 +1,31 @@
-// style go here
+* {
+  box-sizing: border-box;
+  font-family: Arial, sans-serif;
+  background-color: #728fce;
```

Only `box-sizing` property allows for `*` selector
```suggestion
  box-sizing: border-box;
}

body {
  font-family: Arial, sans-serif;
  background-color: #728fce;
}
```